debug/index.html

```diff
@@ -58,7 +58,8 @@
     zoom: 12.5,
     center: [-77.01866, 38.888],
     style: 'mapbox://styles/mapbox/streets-v8',
-    hash: true
+    hash: true,
+    workerCount: 3
```

Comment:
I don't think we need it set in the debug page — let's leave it to rely on `hardwareConcurrency`, which most of the browsers used by GL JS developers support. Or we could add but comment out.